cohttp.install

```diff
@@ -1,7 +1,16 @@
 bin: [
-  "?cohttp-server-async"
-  "?cohttp-server-lwt"
-  "?cohttp-curl-async"
-  "?cohttp-curl-lwt"
-  "?cohttp-proxy-lwt"
+  "?cohttp_server_async.native" { "cohttp-server-async" }
+  "?cohttp_server_async.byte"   { "cohttp-server-async" }
```

Comment:
The opam manual says that files will be installed in sequence. https://opam.ocaml.org/doc/manual/dev-manual.html#sec25